### ISSUE TITLE
Remove System.out.println() from PolyUtil and FactorAbstract

### ISSUE THREAD

**Issue by axkr** [kredel/java-algebra-system#6]
Could you please remove System.out.println() from [PolyUtil#recursiveDivide](https://github.com/kredel/java-algebra-system/blob/master/src/edu/jas/poly/PolyUtil.java) and [FactorAbstract.java](https://github.com/kredel/java-algebra-system/blob/master/src/edu/jas/ufd/FactorAbstract.java)

I'm catching the RuntimeException in my factoring algorithm and don't need the output in Console.
Can you add it to your logger?

**Comment by kredel** [comment]
It will be changed in the next release. println will be changed to logger.warn.

**Comment by axkr** [comment]
For the input` a * c + a * d * x + b * c * x + b * d * x^2` and variables `[a, b, c, d, x]` in Symja I'm running into the output "no exact division:" of `PolyUtil #coefficientPseudoDivide()` or `PolyUtil #coefficientBasePseudoDivide()`. 

Can you throw a `RuntimeException()` in these cases instead of running on.

Stacktrace>:
```
PolyUtil.coefficientPseudoDivide(GenPolynomial<GenPolynomial<C>>, GenPolynomial<C>) line: 1553	
SquarefreeRingChar0<C>.squarefreePart(GenPolynomial<C>) line: 351	
SquarefreeRingChar0<C>(SquarefreeAbstract<C>).isSquarefree(GenPolynomial<C>) line: 99	
FactorInteger<MOD>.factorsSquarefreeHensel(GenPolynomial<BigInteger>) line: 898	
FactorInteger<MOD>.factorsSquarefree(GenPolynomial<BigInteger>) line: 719	
FactorInteger<MOD>(FactorAbstract<C>).factors(GenPolynomial<C>) line: 582	
Algebra$Factor.factor(IAST, List<IExpr>, boolean) line: 1562	
Algebra$Factor.evaluate(IAST, EvalEngine) line: 1529	
EvalEngine.evalASTBuiltinFunction(ISymbol, IAST) line: 686	
```

**Comment by kredel** [comment]
It will be changed in the next release. The RuntimeException will be thrown.

**Comment by kredel** [comment]
Solved in this release 2.6.5918
